frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -44,33 +47,41 @@ const Details_ = ({flags, obj: pvc}) => {
   const storageClassName = _.get(pvc, 'spec.storageClassName');
   const volumeName = _.get(pvc, 'spec.volumeName');
   const requestedStorage = _.get(pvc, 'spec.resources.requests.storage');
-  const storage = _.get(pvc, 'status.capacity.storage');
+  const volumeMode = _.get(pvc, 'status.volumeMode');
   const accessModes = _.get(pvc, 'status.accessModes');
   return <div className="co-m-pane__body">
     <SectionHeading text="PersistentVolumeClaim Overview" />
     <div className="row">
-      <div className="col-sm-6">
-        <ResourceSummary resource={pvc}>
+      <div className="col-sm-4">
+        <ResourceSummary resource={pvc} showPodSelector={false} showNodeSelector={false}>
           <dt>Label Selector</dt>
           <dd><Selector selector={labelSelector} /></dd>
         </ResourceSummary>
       </div>
-      <div className="col-sm-6">
+      <div className="col-sm-4">
         <dl>
           <dt>Status</dt>
-          <dd><PVCStatus pvc={pvc} /></dd>
+          <dd> { volumeName ? <React.Fragment> <PVCStatus pvc={pvc} /> to <ResourceLink kind="PersistentVolume" name={volumeName} /> </React.Fragment> : <PVCStatus pvc={pvc} /> } </dd>
+          <dt>Requested</dt>
+          <dd>{requestedStorage || '-'}</dd>
+          <dt>Used</dt>
+          <dd>{'-'}</dd>
+          {!_.isEmpty(accessModes) && <React.Fragment><dt>Access Modes</dt><dd>{accessModes.join(', ')}</dd></React.Fragment>}
+          <dt>Volume Mode</dt>
+          <dd>{volumeMode ? {volumeMode} : 'Filesystem' }</dd>
```

Comment:
```suggestion
          <dd>{volumeMode || 'Filesystem' }</dd>
```

Reply:
Ack. This seems like a better code suggestion. Thanks!